smqtk_detection/impls/detect_image_objects/centernet.py

```diff
@@ -369,6 +374,24 @@ def _dets_mat_to_list(
         # Create and collect tuples.
         dets_list = []
         for det, cls_idx in zip(dets_mat, class_indices):
+            # If max vertex < min vertex, reorder/reject detection based on config
+            if not (det[2:4] >= det[0:2]).all():
+                if self.reorder_malformed_bboxes:
+                    warnings.warn(
+                        f"Reordering malformed bbox: {det[0:4]}",
+                        RuntimeWarning,
+                    )
```

Comment:
Is this something that is output every time this is encountered / on first encounter? This may be something that more appropriate for debug logging. This is certainly fine as is for now.